### PR TITLE
validate email format before responding to announce

### PR DESCRIPTION
**Pull request by webplate**
fixes the main problem of #157 and is good practice anyway in my opinion.

I reused the simple check of @thb.

maybe we should make a proper function for it. But I don't know where I should put it in a Rail project.

### CODE REVIEW

File: app/models/message.rb [hunk 1]
```diff
@@ -3,7 +3,8 @@ class Message < ApplicationRecord
   belongs_to :trip
 
   validates_presence_of :trip, :sender_name, :sender_email, :body
-
+  validates_format_of :sender_email, with: /\A[\w+\-.]+@[a-z\d\-.]+\.[a-z]+\z/i
```
Comment (nicolasleger) [inline app/models/message.rb]: We could add the Regex in a constant in `ApplicationRecord` or create a custom validator to not repeat the code from `trip.rb`.

**Comment by nicolasleger** [comment]
thanks, I just made a comment, let me know if okay or need help.

File: app/models/message.rb (hunk 1 above)
Comment (webplate) [inline app/models/message.rb]: Hmmm.. I'm not for the regex storage as we could want to futher check the email adress (blacklist or else) the creation of a custom validator seems nice but is something I have to learn.
any clues ? where do I put this validating function ?

Comment (nicolasleger) [inline app/models/message.rb]: You're right! You can place custom valdiators in `app/validators` http://guides.rubyonrails.org/active_record_validations.html#custom-validators

**Comment by webplate** [comment]
Thanks again @nicolasleger ! Seems done to me :)

**Comment by nicolasleger** [comment]
Good! Thanks

**Comment by Poussinou** [comment]
Good PR, we will merge (like other PRs) ASAP :)
Thanks for this

**Comment by AlexBillbao** [comment]
Ca a été mergé sur le site test ou sur le site public "en prod"  ?

**Comment by webplate** [comment]
@AlexBillbao : Dur de savoir, seul @thb a accès au déploiement prod.

**Pull request merged**